squad-show-or-cancel-jobs

```diff
@@ -38,7 +38,14 @@ def arg_parser():
         "--days",
         required=False,
         default=10,
-        help="environment",
+        help="days",
+    )
+
+    parser.add_argument(
```

Comment:
you can add
```
type=int,
```
for both `--days` and `--hours` and avoid `int(args.days)` and `int(args.hours)`

Reply:
oh right doing that. Thank you @chaws